strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/providers/layout/LayoutProviderRegistry.java

```diff
@@ -31,14 +34,18 @@
     @Inject
     private ConfigurationManagementService configurationManagementService;
 
+    @Inject
+    private List<LayoutProvider> layoutProviders;
```

Comment:
Where is this being injected from? Do you mean to say that this is in one of the configuration classes? I don't think I like this idea a lot...

Reply:
No. Spring lookups for all known `LayoutProvider`s in the application context and injects them here.